src/arenaui.cpp

```diff
@@ -288,6 +318,10 @@ void ArenaUI::on_actionOpenArena_triggered()
 
     arenaScene->clear();
     ui->casuTree->clear();
+    ui->groupTree->clear();
+    arenaScene->selectionTreeWidget = new QCasuTreeItem(ui->groupTree,"Selected CASUs");
+    ui->groupTree->addTopLevelItem(arenaScene->selectionTreeWidget);
+    arenaScene->selectionTreeWidget->setHidden(true);
```

Comment:
Unique CASU group for selected CASUs (it has same functionality as CASU group without grouping them)